_sass/components/_high_contrast.scss

```diff
@@ -0,0 +1,325 @@
+#accessibility-nav {
```

Comment:
This file can be split up since it seems to include a lot of layout/component styling, which you have already split up for normal contrast in this ticket

Reply:
I agree - I split most things out into separate files. It may cause a few specificity problems but we can shuffle the order of the `@include` statements to fix that.